projects/packages/my-jetpack/_inc/components/my-jetpack-screen/index.jsx

```diff
@@ -122,6 +129,9 @@ export default function MyJetpackScreen() {
 		return null;
 	}
 
+	const globalNoticeMessage = message ? message : messageDeprecated;
```

Comment:
Strange, I did this at first and it wasn't working. I felt it wasn't worth too much time since this is temporary code but now it seems to work just fine 🤷🏻

Reply:
Sometimes typescript servers might have a hiccup in linting the code 😅